ServiceInnovationLab/pancake-backend
# Cors

# What has changed and why?
Moved CORS into application.rb
# How to test this change
I have tested this locally. Run both front-end and back-end locally, fill in form then check admin for entry creation.
